### PR TITLE
Fix serializing additional issue metadata

### PR DESCRIPTION
**Pull request by alhijazi**

### CODE REVIEW

File: src/clusterfuzz/_internal/datastore/data_handler.py [hunk 1]
```diff
@@ -599,6 +600,10 @@ def get_issue_description(testcase,
 
   # Add additional body text from metadata.
   issue_metadata = testcase.get_metadata('issue_metadata', {})
+  try:
+    issue_metadata = json.loads(issue_metadata)
+  except:
+    pass
```
Comment (jonathanmetzman) [inline src/clusterfuzz/_internal/datastore/data_handler.py:606]: This seems pretty hacky. I think we need to store the metadata field as something other than a string:string map.

Comment (alhijazi) [reply src/clusterfuzz/_internal/datastore/data_handler.py:606]: I will think about a better way to do this, I don't have many ideas as the issue_metadata could be a dictionary of dictionaries

Comment (jonathanmetzman) [reply src/clusterfuzz/_internal/datastore/data_handler.py:606]: I think we need to make the field a json-serialized string instead.

Comment (alhijazi) [reply src/clusterfuzz/_internal/datastore/data_handler.py]: https://github.com/google/clusterfuzz/pull/3915

**Comment by jonathanmetzman** [comment]
Superseded by https://github.com/google/clusterfuzz/pull/4174

**Pull request closed without merging**